pkg/controller/apiclient/apiclient_util_test.go

```diff
@@ -14,18 +14,180 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package apiclient
+package apiclient_test
```

Comment:
@rifung got it. we don't have a generated client right now, so I don't think it's worth starting to build our own reaction chain code right now. let's wait until we can use the generated code to do it. this is fine for now